docs/development/convention.md

```diff
@@ -1,48 +1,108 @@
+```c++
+// DON'T DO:
+// "foo" is 10
+// `foo` is 10
+
+// DO:
+// |foo| is 10
+size_t foo = 10;
+```
+
+[Correlating Google C++ commenting rule here](https://google.github.io/styleguide/cppguide.html#Function_Comments)
+
+#### Greek Letters
+
+Greek letters are often used in explanations of mathematical processes in our comments. We use the "Small" version of Greek unicode letters to standardize  and assist searching capabilites.
+
+For example:
+- Use θ (U+03B8 -- "Greek Small Letter Theta")
+- Do not use Θ (U+0398 -- "Greek Capital Letter Theta")
+
+
+### Build Targets
 
-## Commit
+Since styling Bazel files is not specified in the Google C++ style guide, here are the rules we abide by:
+1. Order all `tachyon_cc_unittest` targets after all `tachyon_cc_library` targets.
+2. Order targets alphabetically within each section.
+3. Name the unittest target name after the pertaining directory the file is included in.
+
+Refer to [`tachyon/base/BUILD.bazel`](https://github.com/kroma-network/tachyon/blob/main/tachyon/base/BUILD.bazel) as an example.
+
+## Logging
+We use `VLOG` messages in our code to show us the progress of the current running processes. We have split `VLOG` into 3 levels:
+
+1. `VLOG(1)` FYI - FOR YOUR INFORMATION
+  Provides insight to basic information of the current process. Examples include:
+     - Proving schemes/elliptic curves/parameters currently in use
+     - Time between steps of processes
+     - Start and end times of processes
+
+1. `VLOG(2)` FOR DEBUGGING
+  Exposes inner progress of the current process for easier debugging. Examples include:
+     - Values of variables such as theta, beta, gamma, challenge, etc
+
+1. `VLOG(3)` FOR TRACING
+  Details outputs designed to help tracing. Examples include:
+     - Output of the proof
+     - Vkey string
+
+## Commits
 
 ### Commit Type Classification
 
 Deciding the commit type can be difficult if the change appears to fall within the definitions of multiple different types. Here are further clarifications on how to determine the proper commit type given previous cases we have come across.
 
-- Any modifications made to files associated with the build process, such as `BUILD.bazel`, are categorized as a **build** commit. This classification includes changes, typo corrections, and formatting as well.
-- **test** commits are for cases where only tests are added or modified. Commits that remove tests are classified as **refac**.
+- Any modifications made to files associated with the build process, such as `BUILD.bazel`, are categorized as a **build** commit. This classification includes changes, typo corrections, and formatting.
+- Commits that add or modify tests are classified as **test** commits, while commits that remove tests are considered **refac** commits.
 
 For more information on our Commit Message Format, see [here](https://github.com/kroma-network/.github/blob/main/CONTRIBUTING.md#commit-message-format).
+
+## GPU
+
+### Naming
```

Comment:
Please refer to https://github.com/DavidAnson/markdownlint/blob/v0.33.0/doc/md024.md

The header is duplicated!